docs/users-guide/08-dashboard-filters.md

```diff
@@ -58,6 +57,24 @@ Once you’ve added a filter to your dashboard, just click on it to select a val
 
 ![Using a filter](images/dashboard-filters/08-use-filter.png)
 
+### Choosing between a dropdown or autocomplete for your filter
+
+As of Metabase 0.29, we've greatly improved how filtering works with fields that have lots of values. If the field you're using for a filter has more than 100 unique values, you'll now automatically see a search box with autocomplete suggestions.
```

Comment:
Not sure we should call out a specific version in the docs? I.e. I should be able to read a given versions docs and get info on that version.